pkg/agent/cniserver/interface_configuration_windows.go

```diff
@@ -504,3 +518,31 @@ func (ic *ifConfigurator) getInterceptedInterfaces(
 func (ic *ifConfigurator) getOVSInterfaceType() int {
 	return internalOVSInterfaceType
 }
+
+func (ic *ifConfigurator) addPostInterfaceCreateHook(containerID, endpointName string, containerAccess *containerAccessArbitrator, hook postInterfaceCreateHook) error {
+	if containerAccess == nil {
+		return fmt.Errorf("container lock cannot be null")
+	}
+	go func() {
+		err := wait.PollImmediate(time.Second, 60*time.Second, func() (done bool, err error) {
+			ifaceName := fmt.Sprintf("vEthernet (%s)", endpointName)
+			containerAccess.lockContainer(containerID)
+			defer containerAccess.unlockContainer(containerID)
+			if _, ok := ic.getEndpoint(endpointName); !ok {
+				klog.InfoS("HNSEndpoint doesn't exist in cache, exit goroutine", "HNSEndpoint Name", endpointName)
+				return true, nil
+			}
+			if !util.HostInterfaceExists(ifaceName) {
+				klog.Errorf("Waiting for interface %s to be created", ifaceName)
```

Comment:
Can we use structured logging and Info since missing interface is expected in some scenario? otherwise user would get error log even the whole workflow works fine.

Reply:
Sure, updated.